src/util/unicode.h

```diff
@@ -25,6 +24,10 @@ std::wstring widen(const std::string &s);
 std::string
 utf32_native_endian_to_utf8(const std::basic_string<unsigned int> &s);
 
+/// \param utf8_str: UTF-8 string
+/// \return UTF-32 encoding of the string
+std::u32string utf8_to_utf32(const std::string &utf8_str);
```

Comment:
Name `_native_endian` for consistency

Reply:
Note commit message -- all functions use native endianness, thus no need make this part of the function name. It may make sense to remove the 'native endianness' everywhere.